frontend-auth-module/projects/sl-auth-lib/src/lib/services/auth.service.ts

```diff
@@ -73,7 +73,12 @@ export class AuthService {
               if (res?.responseCode === "OK") {
                 localStorage.clear();
                 this.sendMessage();
-                this.router.navigate(['/login']);
+                if(this.configData?.intialPage){
```

Comment:
spelling check required based on the config change

Reply:
done